README.md

```diff
@@ -1,7 +1,17 @@
-# MapLibre GL style specification & utilities - Design proposals are required
+# MapLibre GL style specification & utilities
 
 This directory contains code and reference files that define the MapLibre GL style specification and provides some utilities for working with MapLibre styles.
 
+The style specification is used in MapLibre GL JS and in MapLibre GL Native. Our long-term goal is to have feature parity between the web and the native libraries.
+
+## Contributing
+
+If you want to contribute to the style specification, please open an issue with a design proposal. Once your design proposal has been accepted, you can open a pull request and implement your changes.
+
+We aim to avoid breaking changes in the MapLibre style specification, because it makes life easier for our users.
+
+If you want to extend the MVT specification or the TileJSON specification, feel free to open a discussion here in this repository. These two specifications have not been forked so far, but this repo here can serve as a platform for discussion about how to move forward regarding MVT and TileJSON in MapLibre.
+
 ## npm package
```

Comment:
Would be nice to have some links here to npm and some badges about npm version, maybe an icon, IDK...

Reply:
Added